metal/roles/pxe_server/tasks/main.yml

```
@@ -11,10 +11,10 @@
     cmd: "xorriso -osirrox on -indev {{ iso.dest }} -extract / {{ role_path }}/files/data/os"
     creates: "{{ role_path }}/files/data/os/.treeinfo"
 
-- name: Generate DHCP config
+- name: Generate DNSmasq config
```

Nit: same as the above

```suggestion
- name: Generate dnsmasq config
```